app/models/agents/lifx_pulse_lights_agent.rb

```diff
@@ -0,0 +1,64 @@
+module Agents
+  class LifxPulseLightsAgent < Agent
```

Comment:
Maybe just LifxLightsAgent so that as you expand it, it doesn't need to be renamed?